pkg/operators/ustack/ustack.go

```diff
@@ -54,19 +55,44 @@ func (o *Operator) GlobalParams() api.Params {
 }
 
 func (o *Operator) InstanceParams() api.Params {
-	return nil
+	return api.Params{&api.Param{
+		Key:          symbolizersParam,
+		Description:  `Symbolizers to use. Possible values are: "none", "auto", or comma-separated list among: "symtab"`,
+		DefaultValue: "auto",
+	}}
 }
 
 func (o *Operator) InstantiateDataOperator(gadgetCtx operators.GadgetContext, instanceParamValues api.ParamValues) (operators.DataOperatorInstance, error) {
-	s, err := symbolizer.NewSymbolizer()
-	if err != nil {
-		return nil, err
-	}
-
 	instance := &OperatorInstance{
-		symbolizer:    s,
 		subscriptions: make(map[datasource.DataSource][]func(ds datasource.DataSource, data datasource.Data) error),
 	}
+
+	opts := symbolizer.SymbolizerOptions{}
+	symbolizers := instanceParamValues[symbolizersParam]
+	switch symbolizers {
+	case "none":
+		opts.UseSymtab = false
+	case "auto":
+		opts.UseSymtab = true
+	default:
+		list := strings.Split(symbolizers, ",")
```

Comment:
an empty list would result in an error, right? Should we check for empty list and assume it's "none"?

Reply:
Ok. I added "" in the `case "none"` above.